plugin/pkg/scheduler/factory/factory.go

```diff
@@ -136,19 +137,59 @@ func NewConfigFactory(
 	// We construct this here instead of in CreateFromKeys because
 	// ScheduledPodLister is something we provide to plug in functions that
 	// they may need to call.
-	var scheduledPodIndexer cache.Indexer
-	scheduledPodIndexer, c.scheduledPodPopulator = cache.NewIndexerInformer(
-		c.createAssignedNonTerminatedPodLW(),
-		&v1.Pod{},
-		0,
-		cache.ResourceEventHandlerFuncs{
-			AddFunc:    c.addPodToCache,
-			UpdateFunc: c.updatePodInCache,
-			DeleteFunc: c.deletePodFromCache,
+	podInformer.Informer().AddIndexers(cache.Indexers{cache.NamespaceIndex: cache.MetaNamespaceIndexFunc})
```

Comment:
The generated shared informers have the namespace index enabled by default - see e.g. https://github.com/kubernetes/kubernetes/blob/a86fabb9d22b94f61b967510f4fb82dd98946bd5/pkg/client/informers/informers_generated/externalversions/core/v1/pod.go#L56